hub/powertoys/file-actions-menu.md

```diff
@@ -0,0 +1,72 @@
+| PowerRename | Launches the [PowerRename utility](./powerrename.md). | Always |
+| ImageResizer | Launches the [ImageResizer utility](./image-resizer.md). | 3 |
+| File Locksmith | Launches the [File Locksmith utility](./file-locksmith.md). | Always |
+| Unblock | Unblocks from the web downloaded files | 5 |
+| Copy image from clipboard to folder | Copies the image from the clipboard into the selected folder | 1, 6, 7 |
+| Copy image to clipboard | Copies the currently selected image to the clipboard (as image type and not as a file). | 1, 3 |
+| Copy to | Copy the selected files to another location, selectable via a folder browser dalog. | Always |
+| Move to | Move the selected files to another location, selectable via a folder browser dalog. | Always |
+| Save as | Saves the selected file to a new location, selectable via a save as dialog. | 1 |
+| New folder with selection | Creates a new folder with the selected files. | Always |
+| Uninstall | Uninstalls the selected application/the application to which this file belongs. | 4, 8 |
+| Extract images | Extract images from the dll or the exe file into a folder of the same name. | 1, 4 |
+| Merge contents | Merges the contents of the selected files into a new file | 2 |
+| Copy content to clipboard | Copies the content of the selected file into the clipboard. If it is a directory the tree of the directory will be copied into the clipboard. | 1 |
+| Collapse folder structure | Collapses the whole folder structure inside the selected folder | 1, 6 |
+| Copy file content | Copies the file content into the clipboard in a certain way.<br/>**As plaintext**: Copies the whole file as normal text.<br/>**As base64 data link**: Copies the file content as a data url with base64 encoded content. For more information you can read the [specification](https://www.rfc-editor.org/rfc/rfc2397).<br/>**C escaped**: Copies the file content as a C escaped string (`\` → `\\`, newline → `\n\r`, etc.).<br/>**XML escaped**: Copies the content of the file with XML escape sequences (ex. `<` → `&#x3C;`)<br/>**C byte escaped**: Copies the file content, where every character is replaced with the ascii byte sequence (ex. `<` → `\0x3C`).<br/>**URI escaped**: Copies the file content ready to use for an URI (ex. `<` → `%3C`), | 1 |
+
+## Supported hash types
+
+> [!WARNING]
+> Certain hash types are nowadays considered unsafe (like md5 or SHA1). Please inform yourself about the insecurities of the hash types you use.
+
+Following types of hashes are supported.
+
+- CRC32 Decimal
+- CRC32 Hex
+- CRC64 Decimal
+- CRC64 Hex
+- MD5
+- SHA1
+- SHA256
+- SHA384
+- SHA512
+- SHA3-256
+- SHA3-384
+- SHA3-512
+
+## Resolve shortcut
+
+Some actions allow the option to resolve shortcuts. This will target the files that the shortcuts point to instead of the `.ink`-file itsself.
```

Comment:
The include file with PowerToys install info should be added to the end of every page. We get a lot of feedback from people who don't understand that each of these utilities needs to be installed with PowerToys.
```suggestion
Some actions allow the option to resolve shortcuts. This will target the files that the shortcuts point to instead of the `.ink`-file itsself.

[!INCLUDE [install-powertoys.md](../includes/install-powertoys.md)]
```